### PR TITLE
Added a welcome message for user if no items in list, and light styling

### LINKED ISSUE #7: 7. As a user, I want to see a welcoming prompt to add my first item if my list is empty to help me get oriented to how the app works.
AC:

* The list view, when there are no items to display, should show a prompt (e.g., a button) for the user to add their first item
Notes:
* The wireframe is a little short on the "welcoming" part of the story, but otherwise is functionally complete for the purposes of the AC. https://github.com/the-collab-lab/smart-shopping-list/blob/main/_resources/wireframes/list-empty.png

### PR DESCRIPTION
**Pull request by sparsell**
## Description
- Added conditional rendering after checking to see if `data` is an empty array:
-- If there are items in the list, the items will display automatically, if there are no items in the list, a welcome message and a CTA (link styled as a button) are displayed.

- Added styles to Footer.css and App.css
- Created List.css file; added styles to links and created layout per wireframe.
- Added `react-icons` for 'logo' 
- Added individual footer component to both the list view and addItems view so that you can toggle between each view.

## Related Issue

Closes #7

## Acceptance Criteria

- The list view, when there are no items to display, should show a prompt (e.g., a button) for the user to add their first item

Notes:
- The wireframe is a little short on the "welcoming" part of the story, but otherwise is functionally complete for the purposes of the AC. https://github.com/the-collab-lab/smart-shopping-list/blob/main/_resources/wireframes/list-empty.png

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|   ✓ | :link: Update dependencies |
|    | :scroll: Docs              |

### Before

Screen Shot 2022-04-27 at 6 10 35 PM

### After
Screen Shot 2022-04-27 at 6 09 43 PM

## Testing Steps / QA Criteria

1. run `npm install` to add `react-icons`
2. run `npm start`
3. Ensure you do not have a token in your `localStorage` or confirm you are using a token (joining a list) that does not contain any items.
4. If you are creating a new list, you should see the new UI (welcome message and CTA). 
5. If you are joining a list that has items, you will just see the items listed. They will appear at the top of the screen and you will see the newly styled links (List || Add Item) at the bottom of the screen.  
6.  You are also able to toggle between the list view and the add Item view.

### CODE REVIEW

**Review by daniel-sj-hong — APPROVED** [review]
Deleted my token and proceeded to generate a new token. I saw the welcome message (very cute!). Just wondering though, once a user clicks the Add Item button, are they not allowed to go back to the list view? Maybe this is meant for a future feature and I'm thinking ahead again...Otherwise, I love the little robot!!!

**Review by k-schrattenholzer — APPROVED** [review]
I love the styling! 
Was able to delete my token, and saw the welcome message after creating a new List. I'm such a fan of this lil robot icon!
I did have the same question Daniel mentioned about the List link not appearing on the Add Item page - but other than that, this makes the app look so much friendlier & welcoming!

File: src/views/List/List.css [hunk 1]
```diff
@@ -0,0 +1,25 @@
+.div {
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+  justify-content: flex-start;
+  height: calc(100vh - 75px);
```
Comment (lindseyindev) [inline src/views/List/List.css:6]: Nice use and comfortability with the calc function here! ✨ There's a flex-end property on the body element that's causing all of the content to go to the bottom of the page and this is an interesting solution to get the layout you want

File: src/views/List/List.jsx [hunk 2]
```diff
@@ -17,11 +22,29 @@ export default function List({ token }) {
   }, []);
 
   return (
-    <ul>
-      {data.map((listItem, index) => {
-        const { name } = listItem;
-        return <li key={index}>{name}</li>;
-      })}
-    </ul>
+    <section>
```
Comment (lindseyindev) [inline src/views/List/List.jsx:25]: <section> semantic html we love to see it 👏

it's huge for the accessibility of your websites and I've recently realized as a dev how much nicer and easier it is to read html with <section><article> and other semantic containers versus <div>s everywhere

File: src/views/List/List.jsx [hunk 3]
```diff
@@ -17,11 +22,29 @@ export default function List({ token }) {
...
+            {data.map((listItem, index) => {
+              const { name } = listItem;
+              return <li key={index}>{name}</li>;
+            })}
+          </ul>
+        ) : (
+          <>
+            <h1>Smart Shopping List</h1>
+            <div className="icons">
+              <SiProbot className="icon" />
```
Comment (lindseyindev) [inline src/views/List/List.jsx:38]: Again love how comfortable you are with using the right tools here with the UI library and making those decisions to add new things to the app, I haven't personally used React Icons before 🤖 really enjoying seeing your thoughts and the tools you are using

**Review by lindseyindev — APPROVED** [review]
It seems like you both enjoyed working on this issue! Really thorough PR and you also committed the Footer update before I could even blink I like the way you're adding and using CSS modules as well to keep it really clear what that CSS file applies to

**Pull request merged**